src/modules/taxonomy/taxonomyApiInterfaces.ts

```diff
@@ -56,6 +58,11 @@ export interface ResourceWithParentTopics extends Resource {
   parentTopics: ParentTopic[];
 }
 
+export interface SubjectNameTranslation {
```

Comment:
Det finnes tilsvarende for navn på emner og ressurser, men disse settes fra draft-api ved publisering. Men det kan jo tenkes at det kan være aktuellt å gjøre noko med dette på sikt, så kanskje kalle dette TaxNameTranslation eller nokke sånt. Berre et forslag.